layouts/community/ergodox/nstickney/keymap.c

```diff
@@ -0,0 +1,217 @@
+		            UC(0x00CE), UC(0x00C1), UC(0x00C2), UC(0x00D8), UC(0x00B0), UC(0x00A8),
+		__________, UC(0x00D1), UC(0x00DD), UC(0x00C7), UC(0x00D4), UC(0x203D), __________,
+		                        UC(0x00F7), __________, __________, __________, __________,
+		__________, __________,
+		__________,
+		__________, __________, UC(0x00AA)
+	),
+
+	[NUMP] = LAYOUT_ergodox(
+		// left hand
+		__________, __________, KC_P7,      KC_P8,      KC_P9,      KC_PSLS,    __________,
+		__________, __________, KC_P4,      KC_P5,      KC_P6,      KC_PAST,    __________,
+		__________, __________, KC_P1,      KC_P2,      KC_P3,      KC_PMNS,
+		__________, __________, KC_P0,      KC_PCMM,    KC_PDOT,    KC_PPLS,    __________,
+		__________, __________, __________, __________, __________,
+		                                                            __________, __________,
+		                                                                        __________,
+		                                                __________, __________, __________,
+		// right hand
+		__________, __________, KC_P7,      KC_P8,      KC_P9,      KC_PSLS,    __________,
+		__________, __________, KC_P4,      KC_P5,      KC_P6,      KC_PAST,    __________,
+		            __________, KC_P1,      KC_P2,      KC_P3,      KC_PMNS,    __________,
+		__________, __________, KC_P0,      KC_PCMM,    KC_PDOT,    KC_PPLS,    __________,
+		                        __________, __________, __________, __________, __________,
+		__________, __________,
+		__________,
+		__________, KC_PENT,    __________
+	),
+
+	[FCTN] = LAYOUT_ergodox(
+		// left hand
+		KC_ESC,     KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      __________,
+		KC_PSCR,    RGB_TOG,    KC_HOME,    KC_UP,      KC_END,     KC_PGUP,    __________,
+		KC_PAUS,    __________, KC_LEFT,    KC_DOWN,    KC_RGHT,    KC_PGDN,
+		__________, KC_MPRV,    KC_MRWD,    KC_MPLY,    KC_MFFD,    KC_MNXT,    __________,
+		__________, __________, __________, __________, __________,
+		                                                            KC_F11,     __________,
+		                                                                        __________,
+		                                                __________, __________, __________,
+		// right hand
+		__________, KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_SYSREQ,
+		__________, UC(0x2014), KC_LBRC,    KC_RBRC,    S(KC_LBRC), S(KC_RBRC), KC_INS,
+		            KC_LEFT,    KC_DOWN,    KC_UP,      KC_RGHT,    KC_BRIU,    KC_CLR,
+		__________, KC_MSTP,    KC_MUTE,    KC_VOLD,    KC_VOLU,    KC_BRID,    __________,
+		                        __________, __________, __________, __________, __________,
+		__________, KC_F12,
+		__________,
+		__________, __________, __________
+	),
+};
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+
+	set_unicode_input_mode(UC_LNX); // Linux
```

Comment:
While this is fine here.... you could actually move this to `eeconfig_init_user` instead.  That way, it's not running every time. 

(but AVR code won't update it if it's the same value, so it will not do any harm)

Reply:
Better to do things the best way. Thanks!